homeassistant/config_entries.py

```diff
@@ -241,6 +241,9 @@ class OperationNotAllowed(ConfigError):
 }
 
 
+ConfigFlowResult = FlowResult
```

Comment:
When all config flows are migrated, this will be changed to a sub class of `FlowResult` which adds `minor_version` and `version`